src/test/regress/sql/gp_query_id.sql

```diff
@@ -0,0 +1,148 @@
+--
+-- Test the query command identification
+--
+create extension if not exists gp_inject_fault;
+
+-- start_ignore
+drop function if exists sirv_function();
+drop function if exists not_inlineable_sql_func(i int);
+drop table if exists test_data1;
+drop table if exists test_data2;
+drop table if exists t;
+drop table if exists t1;
+drop table if exists t2;
+-- end_ignore
+
+set client_min_messages = notice;
+select gp_inject_fault('all', 'reset', dbid) from gp_segment_configuration;
+
+create or replace function sirv_function() returns text as $$
+declare
+    result1 bool;
+    result2 bool;
+begin
+    create table test_data1 (x int, y int) with (appendonly=true) distributed by (x);
+    create table test_data2 (x int, y varchar) with (appendonly=true) distributed by(x);
```

Comment:
Why do we need two tables and two inserts for each table? Is one table and one insert enough for the test?

Reply:
" Is one table and one insert enough for the test" - generally yes, but having 2 tables allows to test 2 consequnet calls to `ProcessUtility()`, so there is some value in keeping it that way.